app/models/concerns/permitted_queries.rb

```diff
@@ -21,16 +21,23 @@ def initialize(groups, known_roles, is_admin)
   # @return [Array[String]] list of DRUIDs from APOs that this User can view
   def permitted_apos
     query = groups.map { |g| g.gsub(':', '\:') }.join(' OR ')
-    q = 'apo_role_group_manager_ssim:(' + query + ') OR apo_role_person_manager_ssim:(' + query + ')'
-    known_roles.each do |role|
-      q += ' OR apo_role_' + role + '_ssim:(' + query + ')'
+
+    q = ''
+    if is_admin
+      q += '*:*'
```

Comment:
I'm assuming you checked to make sure the request handler is okay with *:* ?    I think it's okay for e/dismax but can't remember if it's ok for lucene??